src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/FeatureFlag/index.html.twig

```diff
@@ -24,73 +24,81 @@
  *#}
 {% extends '@PrestaShop/Admin/layout.html.twig' %}
 
-{% form_theme stableFeatureFlagsForm '@PrestaShop/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig' %}
-{% form_theme betaFeatureFlagsForm '@PrestaShop/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig' %}
+{% if stableFeatureFlagsForm %}
+  {% form_theme stableFeatureFlagsForm '@PrestaShop/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig' %}
+{% endif %}
+{% if betaFeatureFlagsForm %}
+  {% form_theme betaFeatureFlagsForm '@PrestaShop/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig' %}
+{% endif %}
```

Comment:
You can replace all this in the `FeatureFlagListType` you can use the new `form_theme` option that allows setting this form theme directly in the form options (it didn't exist when this page was created).

Similar to this:
https://github.com/PrestaShop/PrestaShop/blob/e74a2e07451d73b00f5056561ac16adece626097/src/PrestaShopBundle/Form/Admin/Sell/Product/Specification/FeaturesType.php#L69

This way the twig gets much simpler.

Reply:
If you think it's not useful I'll approve the PR it's not blocking